CleverTapSDK/ProductExperiences/CTVar.m

```diff
@@ -84,6 +93,11 @@ - (void)update {
     
     if (![_value isEqual:oldValue]) {
         _hasChanged = YES;
+        
+        // Update _fileURL with new value if it has changed.
+        if ([_kind isEqualToString:CT_KIND_FILE]) {
```

Comment:
File needs to be downloaded ad-hoc if the variable is registered after the initial fetch and applyDiffs